### PR TITLE
Dabarger/local search api

### PR DESCRIPTION
**Pull request by davidbargeron**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR - https://github.com/Azure/azure-rest-api-specs/pull/4349
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [NA] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.ps1 file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please add link to your swagger RP in description.
Once the swagger RP is merged, the SDK needs to be regenerated again to pick up the latest/merged swagger.

Any reason tests are not part of this SDK?
If you cannot add tests, please create an issue and describe the plan to add tests and assign the issue to your team.
Also as this is your first version, we advice you go as -preview, in this case go as 0.9.0-preview. We have quite often seen RPs going as stable as their first version end up introducing breaking changes due to things getting missed or pure customer feedback and end up releasing multiple major version upgraded within a period of month.
one such example
https://www.nuget.org/packages/Microsoft.Azure.Management.StorageSync/
The above RP has now understood and have decide to use -preview as their next release.

File: src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.CognitiveServices.Search.LocalSearch</PackageId>
+		<Description>Provides ability to search for businesses</Description>
+		<Version>0.9.0-preview</Version>
+		<AssemblyName>Microsoft.Azure.CognitiveServices.BingLocalSearch</AssemblyName>    
+		<PackageTags>CognitiveServices;Bing;BingLocalSearch;LocalSearch;Local Search</PackageTags>
+		<PackageReleaseNotes>
+			<![CDATA[
+                >> put release notes here
```
Comment (yangyuan) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj]: Missing release notes?

File: src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		</PackageReleaseNotes>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+	</PropertyGroup>
+
+	<!-- Please do not move/edit code below this line -->
+	<Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
+	<ItemGroup>
```
Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj]: delete line 25 to 28

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please address the existing comments.
Still missing release notes and one minor edit

File: src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch.Tests/BingLocalSearch.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```
[the remark this answers is not in the document]
Comment (davidbargeron) [reply src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch.Tests/BingLocalSearch.Tests.csproj]: https://github.com/Azure/azure-sdk-for-net/pull/4962/commits/648c1186fce4ddc4870e8990bf58de5d3df491ad

File: src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj [hunk 4]
```diff
@@ -0,0 +1,93 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (davidbargeron) [reply src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj]: https://github.com/Azure/azure-sdk-for-net/pull/4962/commits/648c1186fce4ddc4870e8990bf58de5d3df491ad

File: src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/packages.config [hunk 5]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (davidbargeron) [reply src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/packages.config]: https://github.com/Azure/azure-sdk-for-net/pull/4962/commits/648c1186fce4ddc4870e8990bf58de5d3df491ad

File: src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/generate.cmd [hunk 6]
```diff
@@ -0,0 +1,7 @@
+::
```
[the remark this answers is not in the document]
Comment (davidbargeron) [reply src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/generate.cmd]: https://github.com/Azure/azure-sdk-for-net/pull/4962/commits/648c1186fce4ddc4870e8990bf58de5d3df491ad

File: src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj (hunk 1 above)
Comment (davidbargeron) [reply src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj]: https://github.com/Azure/azure-sdk-for-net/pull/4962/commits/821162637d4dc9db162df81b71a02abf6b974fae

File: src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj (hunk 2 above)
Comment (davidbargeron) [reply src/SDKs/CognitiveServices/dataPlane/Search/BingLocalSearch/BingLocalSearch/BingLocalSearch.csproj]: https://github.com/Azure/azure-sdk-for-net/pull/4962/commits/821162637d4dc9db162df81b71a02abf6b974fae

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**